src/lib/blueprint.ts

```diff
@@ -285,19 +282,15 @@ const createParameters = (operation: OpenapiOperation): Parameter[] => {
 }
 
 const createParameter = (param: OpenapiParameter): Parameter => {
+  const validatedParam = ParameterSchema.parse(param)
```

Comment:
If we are going to use a prefix, then it should be `parsed` not `validated`.

```suggestion
  const parsedParam = ParameterSchema.parse(param)
```